### PR TITLE
Fix title case strings

### PR DESCRIPTION
**Pull request by marusak**

### CODE REVIEW

File: pkg/storaged/stratis-details.jsx [hunk 1]
```diff
@@ -364,17 +364,17 @@ export const StratisPoolDetails = ({ client, pool }) => {
 
     function create_fs() {
         dialog_open({
-            Title: _("Create Filesystem"),
+            Title: _("Create filesystem"),
```
Comment (martinpitt) [inline pkg/storaged/stratis-details.jsx:367]: If we break translatable strings, then let's please fix them properly, and move to "file system".

**Review by martinpitt — CHANGES_REQUESTED** [review]
Always a little sad to break so many translations -- but I suppose we can't update our po/* files, we'd have to do that in weblate now?

File: pkg/storaged/stratis-details.jsx (hunk 1 above)
Comment (marusak) [reply pkg/storaged/stratis-details.jsx:367]: we use "file system" once in the whole Cockpit and "filesystem" dozens of times. Are you sure I should change this one occurrence to "file system"?

Comment (martinpitt) [reply pkg/storaged/stratis-details.jsx:367]: IMHO *if* we touch a translatable string, we should better just fix it at the same time.

Comment (marusak) [reply pkg/storaged/stratis-details.jsx]: Done

**Review by allisonkarlitskaya — CHANGES_REQUESTED** [review]
"filesystem" is very consistently used through (for example) `man mount(8)`

**Review by allisonkarlitskaya — DISMISSED** [review]
...actually, I changed my mind.  I prefer "file system" now!!! :rofl:

In seriousness, though, you'll need to rebase this

Long story short: I didn't notice that Martin was checking out the head commit when I removed his use of a rebasing jumpstart (and this was before we properly supported unmerging).  I'll fix that in a separate issue.  It basically means that we use the merged workflow with the head tree, and in your case, that doesn't include the `test` script.  So ya... rebase please :/

**Comment by marusak** [comment]
Re/plant-based

**Comment by martinpitt** [comment]
Still two pixel test errors, sorry

**Comment by martinpitt** [comment]
Still five more pixel tests to update, I force-pushed them now. Thanks!

**Review by martinpitt — APPROVED** [review]

**Review by martinpitt — APPROVED** [review]

**Pull request merged**